### DISCUSSION TITLE
Word Art

### DISCUSSION THREAD

**Discussion by christopherblaser** [dolanmiu/docx#2431] [Ideas / Requests]
Hello there,

Is it possible to patch into a word art element?

**Comment by dolanmiu** [comment]
Not at the moment sorry, way out of scope

Word Art is specific to Microsoft Word btw. For example, LibreOffice and doesn't have such concept, and they have their own similar one called `FontWorks`

Apple Pages doesn't even have this feature entirely: https://discussions.apple.com/thread/2553766?sortBy=best

if `docx` were to implement this feature, then it would probably be custom and be a big project

Edit: Misread your question

But due to the nature of WordArt being a vector graphic, then probably not
